.buildkite/hooks/pre-command

```diff
@@ -53,3 +53,8 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-agent-binary-dra" ]]; then
     release_manager_login
   fi
 fi
+
+# BUILDKITE_PIPELINE_SLUG should match elastic-agent for PRs, and elastic-agent-extended-tests once it has merged to main
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-agent"* && ("$BUILDKITE_STEP_KEY" == "integration-cloud-image" || "$BUILDKITE_STEP_KEY" == "integration-fips-cloud-image") ]]; then
+    docker_login
+fi
```

Comment:
@v1v, Do I still need this for the PR?

Reply:
You can now enhance your Buildkite pipeline by explicitly configuring the `integration-cloud-image` step to utilise the plugin for preparing the docker environment. This adjustment will streamline your setup and reduce the black box magic with the `pre-command` hook.

 https://github.com/elastic/elastic-agent/pull/8444 introduced that change. Let me know if you need further assistance